src/app/pages/profile/transaction-information/transaction-information.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, OnInit } from '@angular/core';
+import { TransactionInformationService } from '../../../services/profile/transaction-information/transaction-information.service';
+import { APPCONFIG } from 'src/app/config';
+import { environment } from 'src/environments/environment';
+
+@Component({
+  selector: 'app-transaction-information',
+  templateUrl: './transaction-information.component.html',
+  styleUrls: ['./sass/transaction-information.component.scss']
+})
+export class TransactionInformationComponent implements OnInit {
+  base_url = APPCONFIG.base_url;
```

Comment:
What do you think of getting rid of this line
```
  base_url = APPCONFIG.base_url;
```
Im not seeing where its being used, since you already have `transacationUrl`

Reply:
I worked on that @Tevinthuku